test/testdata/fixtures/single-dc/k8ssandra.yaml

```diff
@@ -4,13 +4,13 @@ metadata:
   name: test
 spec:
   cassandra:
-    serverVersion: 3.11.11
+    serverVersion: 4.0.6
```

Comment:
I thought we were avoiding C* 4.0 in e2e tests?

Reply:
Right, I can't remember precisely why I moved this one to 4.0.6 but know that it's been a while since I've wanted to re-introduce it for at least one of the e2e tests.